src/schemas/f142/f142_rw.cxx

```diff
@@ -149,11 +149,7 @@ uint64_t FlatbufferReader::timestamp(Msg const &msg) const {
 }
 
 FlatbufferReaderRegistry::Registrar<FlatbufferReader>
-    g_registrar_FlatbufferReader(fbid_from_str("f142"));
-
-FileWriter::HDFWriterModule::ptr HDFWriterModule::create() {
-  return FileWriter::HDFWriterModule::ptr(new HDFWriterModule);
-}
+    RegisterEPICSDataReaderModule("f142");
```

Comment:
Would e.g. `RegisterFlatbufferReaderModule` a less confusing name? The casual reader could think that we might do something with the EPICS API here.

Should be `static`.

Reply:
I agree that the name could probably be improved but I strongly disagree that there is a confusion with the EPICS API here as the given types makes it clear that this is not the case. I do not think that `RegisterFlatbufferReaderModule` is an improvement as every single reader (and writer) is a flatbuffer reader/writer. As for static, I will have that fixed in the next commit.

Reply:
Naming things is hard, right? How about giving up and simply having the line be:
```
static HDFWriterModuleRegistry::Registrar<HDFWriterModule> Register("f142");
```
Most of the information we are interested in is already in that line anyways.

Reply:
I like that.

Reply:
Due to name collision, the line is instead:
```
static HDFWriterModuleRegistry::Registrar<HDFWriterModule> RegisterWriter("f142");
```